src/js/utils/user.js

```diff
@@ -7,7 +7,11 @@ export function isChannelLinked(clients, channelType) {
 
 export function getDisplayName(clients, channelType) {
     const client = clients.find((client) => client.platform === channelType);
-    return client && client.displayName;
+    if (channelType === 'twilio') {
```

Comment:
Hmmm maybe the backend should just be returning the correct `displayName` for Twilio?